docs/pages/howto-verabridge/verabridge.mdx

```diff
@@ -73,19 +69,19 @@ Choose the admin address, which will own the bridge. By default, this is your Ow
 
 ### Select Remote Chains
 
-Choose the chains you want to bridge to. For this example, we’ll select OP Sepolia (11155420) and Sepolia (11155111). To add more chains, click "+ Add New Chain." Once you've made your selections, click "Submit Selected Chains" to proceed.
+Choose the chains you want to bridge to. For this example, we will select "OP Sepolia (11155420)" and "Sepolia (11155111)". To add more chains, click "+ Add New Chain." Once you have made your selections, click "Submit Selected Chains" to proceed.
 
-<Screenshot src={selectRemoteNetworks} alt="Select Remote Chains" />
+<Screenshot src={selectRemoteNetworks} alt="Select Remote Networks" />
 
 ### Confirmation
 
 <Screenshot src={complete} alt="Confirmation" />
 
-After submitting the chains, you'll see a confirmation message along with the Token Router Address for each selected chain. To learn more about the deployed contract, copy its address and paste it into the search bar of the relevant blockchain explorer. For example, here's the [deployed contract on Base Sepolia](https://sepolia.basescan.org/address/0xB68C4B24c8a8A2Ee6fECBcf011e817Fef776FF02).
+After select the chains, you will see a confirmation message and the Token Router address of each chain. To learn more about the deployed contract, copy its address and paste it into the search bar of the relevant blockchain explorer. For example, here's the [deployed contract on Base Sepolia](https://sepolia.basescan.org/address/0xB68C4B24c8a8A2Ee6fECBcf011e817Fef776FF02).
 
 <Screenshot src={basescan} alt="Explorer" />
 
-Click "View on VeraBridge" to access the Bridge UI. Connect the wallet holding the token you want to bridge. You can now bridge the token to the selected chains, where a synthetic token will be minted. Similarly, you can bridge the synthetic token back to the original chain as needed.
+Go to the Bridge UI ("View on Verabridge"). Connect a wallet, and try to bridge your token. Once it is on other chains, you can bridge the token back to the original chain as needed.
```

Comment:
*Once it is on another chain

(just to be specific that you can only bridge back from a chain where somewhere previously bridged to, since initial syntehtic supplies are 0)